app/models/region.rb

```diff
@@ -1,8 +1,7 @@
 class Region < ActiveRecord::Base
   has_many :region_bird_links
   has_many :birds, through: :region_bird_links
-  has_many :region_trail_links
-  has_many :trails, through: :region_trail_links
+  has_many :trails
```

Comment:
@cchchoi1986 Take a look at this. Trail and Region have one-to-many relationship, not many-to-many, so we changed it.